casper/src/main/scala/coop/rchain/casper/genesis/contracts/ProofOfStake.scala

```diff
@@ -26,6 +27,7 @@ final case class ProofOfStake(
       "quarantineLength"         -> quarantineLength,
       "numberOfActiveValidators" -> numberOfActiveValidators,
       "posMultiSigPublicKeys"    -> ProofOfStake.publicKeys(posMultiSigPublicKeys),
+      "genesisPosVaultPubKey"    -> ProofOfStake.publicKeys(List(genesisPosVaultPubKey)),
```

Comment:
You can not use `ProofOfStake.publicKeys` . `genesisPosVaultPubKey` only needs one key.

Let's supposed the `genesisPosVaultPubKey` is `0x123456`
The final goal is the render the template from
```
createUnfVault!(*posVaultCh, "$$genesisPosVaultPubKey$$") |
```

to 
```
createUnfVault!(*posVaultCh, "0x123456") |
```

Reply:
We don't need `hexToBytes` here.